test/babashka/cli_test.cljc

```diff
@@ -59,7 +59,8 @@
                  :cause :coerce
                  :msg "Coerce failure: cannot transform input \"dude\" to long"
                  :option :b
-                 :value "dude"}
+                 :value "dude"
+                 :spec nil}
```

Comment:
I went back and forth between conditionally merging `:spec` in and always merging it in. I settled on this so that the keys would be more consistent but I'm still on the fence.

Reply:
Either way is fine with me